infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -251,9 +261,11 @@ async function setHyperchainMetadata() {
             feeReceiverAddress = keyResults.feeReceiver;
         }
     } else {
+        useMatterlabsGeth = true;
         l1Rpc = 'http://localhost:8545';
```

Comment:
@artmakh shouldn't we also change this one?

Reply:
Described in ticket